### PR TITLE
SVG LionWeb logo

### PR DESCRIPTION
**Pull request by enikao**

### CODE REVIEW

File: resources/LangDev Talk.pptx
Comment (dslmeinte) [inline resources/LangDev Talk.pptx:1]: Should this file be here, in this PR/branch?

File: resources/logo-path.svg
Comment (dslmeinte) [inline resources/logo-path.svg:1]: What's the difference between path and non-path?

File: resources/LangDev Talk.pptx
Comment (enikao) [reply resources/LangDev Talk.pptx:1]: Not really ... This was what I started with, and created lots of other stuff for it. I'll take it out.

File: resources/logo-path.svg
Comment (enikao) [reply resources/logo-path.svg:1]: non-path contains the text as text, `path` are pure shapes. The latter is safer for display, as it doesn't rely on the font being available.

Comment (dslmeinte) [reply resources/logo-path.svg:1]: Maybe mention this in a `README.md` in that directory?

**Pull request merged**